cli/util/util.go

```diff
@@ -257,3 +258,25 @@ func ReadEmbedFileBinary(fs embed.FS, path string) ([]byte, error) {
 	}
 	return content, nil
 }
+
```

Comment:
Add comment. Something like this:
// AskConfirm asks the user for confirmation and returns true if yes.

Reply:
Added